### DISCUSSION TITLE
Frequent switching in the i18n list will occur errors

### DISCUSSION THREAD

**Discussion by DrAugus** [vuepress/core#961] [Q&A] [answered]
## Frequent switching in the i18n list will occur errors

![image](https://user-images.githubusercontent.com/34206897/174571674-00d7213f-b5d3-40bb-abb8-d13ccc3bc714.png)

### dev env

![image](https://user-images.githubusercontent.com/34206897/174572874-fbd07af2-5d06-4b99-9aff-8c03202c8f5d.png)

## Repo

[here](https://github.com/DrAugus/draugus.github.io)

## Output

yarn run v1.22.19
$ vuepress-vite dev docs
vuepress-theme-hope:  ‼ Locale /i18n/en/ is missing it's i18n config
vuepress-theme-hope:  ‼ Locale /i18n/fr/ is missing it's i18n config
vuepress-theme-hope:  ‼ Locale /i18n/de/ is missing it's i18n config
vuepress-theme-hope:  ‼ Locale /i18n/it/ is missing it's i18n config
vuepress-theme-hope:  ‼ Locale /i18n/kr/ is missing it's i18n config
backToTop:  ‼ Locale /i18n/en/ is missing it's i18n config
backToTop:  ‼ Locale /i18n/fr/ is missing it's i18n config
backToTop:  ‼ Locale /i18n/de/ is missing it's i18n config
backToTop:  ‼ Locale /i18n/it/ is missing it's i18n config
backToTop:  ‼ Locale /i18n/kr/ is missing it's i18n config
copy-code:  ‼ Locale /i18n/en/ is missing it's i18n config
copy-code:  ‼ Locale /i18n/fr/ is missing it's i18n config
copy-code:  ‼ Locale /i18n/de/ is missing it's i18n config
copy-code:  ‼ Locale /i18n/it/ is missing it's i18n config
copy-code:  ‼ Locale /i18n/kr/ is missing it's i18n config
md-enhance:  ‼ Locale /i18n/en/ is missing it's i18n config
md-enhance:  ‼ Locale /i18n/fr/ is missing it's i18n config
md-enhance:  ‼ Locale /i18n/de/ is missing it's i18n config
md-enhance:  ‼ Locale /i18n/it/ is missing it's i18n config
md-enhance:  ‼ Locale /i18n/kr/ is missing it's i18n config
info Initializing VuePress and preparing data...
reading-time:  ‼ Locale /i18n/en/ is missing it's i18n config
reading-time:  ‼ Locale /i18n/fr/ is missing it's i18n config
reading-time:  ‼ Locale /i18n/de/ is missing it's i18n config

**Comment by Mister-Hope** [comment, marked as answer]
I think this probably is a theme bug, not a vuepress one, please report your problems in the right place.

**Comment by Mister-Hope** [comment]
Also, as the author of vuepress-theme-hope, I debug your project for a couple of minutes, and I can not reproduce your issue on my template. also your project is providing its own i18n config, and register some components itself.

So if you want to report it, you'd better make a mininal reproduction repo starting from the template, to make sure the issue is not on your side.

**Reply by DrAugus** [reply]
得频繁切换语言才能坏，正常切换不会报错的 :(
